### DISCUSSION TITLE
Elastic FWI for OBN data

### DISCUSSION THREAD

**Discussion by scott314159** [slimgroup/JUDI.jl#248] [Q&A]
I have some 4C (P, X, Y, Z) OBN field data.  I've extracted a 2D(ish) line and I want to see if I can get an FWI update but I am not sure how to deal with the source reciprocity aspect of this.

Given that I have many more sources than receivers (especially when I go to 3D) and I want to get this as efficient as possible I would like to use reciprocity to do the modeling (use the receiver position as the source).  In order to do this I will switch the SourceX and GroupX locations which is easy enough but then there is the matter of specifying the wavelet.  I think there are several issues that need to be addressed:
1) I am at the water/solid (ocean bottom) interface.  I was thinking of starting with a simple wavelet for the pressure component but wondering if anyone has a different experience for the pressure component wavelet specification?
2) I also have the 3C geophones that I want to model and then invert for (I figure modeling to see if I am anywhere close to the field data is a good start).  Is there a way to specify a vector source (source with directionality e.g. up/down for the vertical component, North for Horizontal component 1, etc.) within the existing Judi framework?
3) then there is the more mundane question of how to input the data into the inversion.  Does Judi expect a regular grid of data (same number of traces/gather) and on a regular grid or will it take an irregular set of data?

Thanks

**Comment by mloubout** [comment]
Hi

Thank you for your interest in JUDI; I hope you can find a good use for it.


1. I think it's the standard way to do it, yes and if I remember correctly, you need to inject in VZ, not pressure after reciprocity, but I would double-check the literature
2. The elastic capabilities or JUDI are currently limited to forward modeling of pressure data only. Adding support for inversion and multi-component is on the roadmap, but I unfortunately do not have much free time to spend on it right now, so I cannot put a timeline on it.
3. JUDI is designed to support field data and does not expect any regularity for the shot records, you can have each individual shot experiment with different number of trace, time sampling and number of time samples if wanted.

Hope this helps

**Reply by scott314159** [reply]
Thank you.
